core/renderers/common/renderer.js

```diff
@@ -55,6 +63,7 @@ Blockly.blockRendering.Renderer = function() {
 Blockly.blockRendering.Renderer.prototype.init = function() {
   this.constants_ = this.makeConstants_();
   this.constants_.init();
+  this.injectCSS_(this.getCSS_(this.constants_));
```

Comment:
You don't need to pass in the constants object since the function is on the renderer and can access it directly.

Reply:
This was so ``this.constants_`` can remain private, making it protected would also resolve this. It just depends on how we see the constants object being used by subclass renderers.I see everything seems to use getConstants. so perhaps that's the solution. Have getCSS_ use ``getConstants``.

Reply:
Done.